docs/src/inverse_problems/structural_identifiability.md

```diff
@@ -0,0 +1,168 @@
+```example si1
+assess_identifiability(gwo; measured_quantities=[:M], known_p=[:pₑ], loglevel=Logging.Error)
+```
+Not only does this turn the previously non-identifiable `pₑ` (globally) identifiable (which is obvious, given that its value is now known), but this additional information improve identifiability for several other network components.
+
+To, in a similar manner, indicate that certain initial conditions are known is a work in progress. Hopefully the feature should be an available in the near future.
+
+### Providing non-trivial measured quantities
+Sometimes, we are not actually measuring species species, but rather some combinations of species (or possibly parameters). To account for this, `measured_quantities` accepts any algebraic expressions (and not just single species). To form such expressions, species and parameters have to first be `@unpack`'ed from the model. Say that we have a model where an enzyme ($E$) is converted between an active and inactive form, which in turns activates the production of a product, $P$:
+```example si2
+using Catalyst, StructuralIdentifiability # hide
+enzyme_activation = @reaction_network begin
+    (kA,kD), Eᵢ <--> Eₐ
+    (Eₐ, d), 0 <-->P
+end
+```
+and we can measure the total amount of $E$ ($=Eᵢ+Eₐ$), as well as the amount of $P$, we can use the following to assess identifiability:
+```example si2
+@unpack Eᵢ, Eₐ = enzyme_activation
+assess_identifiability(enzyme_activation; measured_quantities=[Eᵢ+Eₐ, :P], loglevel=Logging.Error)
+nothing # hide
+```
+
+### Assessing identifiability for specified quantities only
+By default, StructuralIdentifiability assesses identifiability for all parameters and variables. It is, however, possible to designate precisely which quantities you want to check using the `funcs_to_check` option. This both includes selecting a smaller subset of parameters and variables to check, or defining customised expressions. Let us consider the Goodwind from previously, and say that we would like to check whether the production parameters ($pₘ$, $pₑ$, and $pₚ$) and the total amount of the three species ($P + M + E$) are identifiable quantities. Here, we would first unpack these (allowing us to form algebraic expressions) and then use the following code:
+```example si1
+@unpack pₘ, pₑ, pₚ, M, E, P = goodwind_oscillator
+assess_identifiability(goodwind_oscillator; measured_quantities=[:M], funcs_to_check=[pₘ, pₑ, pₚ, M + E + P], loglevel=Logging.Error)
+nothing # hide
+```
+
+### Probability of correctness
+The identifiability methods used can, in theory, produce erroneous results. However, it is possible to adjust the lower bound for the probability of correctness using the argument `p` (by default set to `0.99`, that is, at least a $99\%$ chance of correctness). We can e.g. increase the bound through:
+```example si2
+assess_identifiability(goodwind_oscillator; measured_quantities=[:M], p=0.999, loglevel=Logging.Error)
```

Comment:
```suggestion
assess_identifiability(goodwind_oscillator; measured_quantities=[:M], p=0.999, loglevel=Logging.Error)
```
Do we have to use `p`? This seems confusing given it usually means parameters in SciML. Maybe use `identifiability_probability` or `ident_prob`?

Reply:
It might be worth an issue there. I think they should consider deprecating that and adding a more descriptive keyword. Can't the `p` keyword be used to pass parameters in SciML problems even? If so it probably shouldn't be used in SI if they are moving to be more SciML compatible.

Reply:
We do use `p` but if you decide to use a more descriptive name, this sounds good to me. Normally, there is much need in changing this parameter anyway.

Reply:
OK, maybe @TorkelE we can add a more descriptive keyword here and translate it to `p` for them internally, and then once SI gets updated with that keyword we can just switch to passing it through?

Reply:
Maybe we should name it `pvalue`.

Reply:
I think in stats everyone calls it a `pvalue` though.

Reply:
well I guess it's not quite a pvalue? It's more of a `prob_threshold`?

Reply:
Yeah, I realized that was what you meant by `pvalue` right after I suggested `probvalue`. I think it is probably ok then.

Reply:
I personally like `prob_threshold` better.

Reply:
I will do prob_threshold . People won't really use it anyway, so a long keyword is not a problem.